great_expectations/expectations/metrics/util.py

```diff
@@ -68,7 +68,7 @@
 
 try:
     import databricks.sqlalchemy as sqla_databricks
-except ImportError:
+except (ImportError, AttributeError):
     sqla_databricks = None  # type: ignore[assignment]
 
```

Comment:
I think this module should probably use our `NotImported` pattern. But I don't want to do that as part of this PR.